packages/executors/http/src/index.ts

```diff
@@ -242,7 +242,26 @@ export function buildHTTPExecutor(
         if (typeof result === 'string') {
           if (result) {
             try {
-              return JSON.parse(result);
+              const parsedResult = JSON.parse(result);
+              if (
+                parsedResult.data == null &&
```

Comment:
These need to check for undefined as well. E.g. the response is `{notGraphql: true}`

Reply:
`== null` should check for both undefined and null

Reply:
👍 woops
I'm too used to strict typescript rules. No `==` for me :)